.gitignore

```diff
@@ -0,0 +1,3 @@
+pointer-constraints-unstable-v1.xml
+relative-pointer-unstable-v1.xml
+
```

Comment:
Let's get rid of .gitignore, it's not needed.

Instead, we can add a `rm pointer-constraints-unstable-v1.xml relative-pointer-unstable-v1.xml` line at the end of https://github.com/go-gl/glfw/wiki/Development#generating-wayland-sourceheader-files and the repo will be clean.